.github/workflows/trigger_jenkins.yaml

```diff
@@ -0,0 +1,30 @@
+name: Trigger Jenkins Job
+
+on:
+  push:
+    branches:
+      - next**
+
+jobs:
+  trigger-jenkins:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Determine Jenkins Job Name
+        id: job_name
+        run: |
+          if [[ "${{ github.ref_name }}" == "next" ]]; then
+            echo "JOB_NAME=scylla-master/next-machine-image-trigger" >> $GITHUB_ENV
+          else
+            VERSION=$(echo "${{ github.ref_name }}" | cut -d'-' -f2)
+            echo "JOB_NAME=scylla-${VERSION}/next-machine-image-trigger" >> $GITHUB_ENV
+          fi
+
+      - name: Trigger Jenkins Job
+        env:
+          JENKINS_USER: ${{ secrets.JENKINS_USERNAME }}
+          JENKINS_API_TOKEN: ${{ secrets.JENKINS_TOKEN }}
+          JENKINS_URL: "https://jenkins.scylladb.com"
+        run: |
+          echo "Triggering Jenkins Job: $JOB_NAME"
+          curl -X POST "$JENKINS_URL/job/$JOB_NAME/build" \
```

Comment:
you can try to add `--fail` flag in curl command that can help to catch errors and in case of failure to send slack msg